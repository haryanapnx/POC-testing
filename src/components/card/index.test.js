import * as React from "react";
import { render, screen } from "@testing-library/react";

import MyCard from "./index";

describe("Card", () => {
  test("renders correctly", () => {
    render(<MyCard>content</MyCard>);
    const card = screen.getByText("content");

    expect(card).toHaveTextContent("content");
  });

  test("Snapshot", () => {
    const { container } = render(<MyCard>content</MyCard>);

    expect(container).toMatchSnapshot();
  });
});
