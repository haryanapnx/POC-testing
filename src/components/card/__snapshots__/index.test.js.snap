// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Snapshot 1`] = `
<div>
  <div
    class="card"
  >
    content
  </div>
</div>
`;
